OpenRA.Test/OpenRA.Test.csproj

```diff
@@ -25,27 +25,39 @@
     <Reference Include="Eluant, Version=1.0.5229.27703, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\thirdparty\Eluant.dll</HintPath>
-      <Private>False</Private>
+      <Private>True</Private>
```

Comment:
No it doesn't work in Windows locally neither. Every compiles fine and everything but when trying to execute the tests with NUnit I get the following error (the same as appVeyor):

Test 'M:PathfinderTests.PathfinderTests.FindPathOnRoughTerrainTest' failed: No se puede cargar el archivo o ensamblado 'OpenRA.Mods.Common, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null' ni una de sus dependencias. El sistema no puede encontrar el archivo especificado.
	System.IO.FileNotFoundException: System.IO.FileNotFoundException: Could not load file or assembly 'OpenRA.Mods.Common, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'.  The system cannot find the file specified.
	File name: 'OpenRA.Mods.Common, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'
	   en System.Reflection.RuntimeAssembly.GetExportedTypes(RuntimeAssembly assembly, ObjectHandleOnStack retTypes)
	   en System.Reflection.RuntimeAssembly.GetExportedTypes()
	   en NUnit.Core.CoreExtensions.InstallAdhocExtensions(Assembly assembly)
	   en NUnit.Core.Builders.TestAssemblyBuilder.Load(String path)
	   en NUnit.Core.Builders.TestAssemblyBuilder.Build(String assemblyName, Boolean autoSuites)
	   en NUnit.Core.Builders.TestAssemblyBuilder.Build(String assemblyName, String testName, Boolean autoSuites)
	   en NUnit.Core.TestSuiteBuilder.BuildSingleAssembly(TestPackage package)
	   en NUnit.Core.TestSuiteBuilder.Build(TestPackage package)
	   en NUnit.AddInRunner.NUnitTestRunner.run(ITestListener testListener, Assembly assembly, ITestFilter filter)
	   en NUnit.AddInRunner.NUnitTestRunner.runMethod(ITestListener testListener, Assembly assembly, MethodInfo method)
	   en NUnit.AddInRunner.NUnitTestRunner.MemberRun.Run(NUnitTestRunner testRunner, ITestListener testListener, Assembly assembly)
	   en NUnit.AddInRunner.NUnitTestRunner.run(ITestListener testListener, Assembly assembly, IRun run)
	   en NUnit.AddInRunner.NUnitTestRunner.RunMember(ITestListener testListener, Assembly assembly, MemberInfo member)
	   en TestDriven.TestRunner.AdaptorTestRunner.Run(ITestListener testListener, ITraceListener traceListener, String assemblyPath, String testPath)
	   en TestDriven.TestRunner.ThreadTestRunner.Runner.Run()